api/templates/emails/campCollaborationInvite.de.text.twig

```diff
@@ -1,9 +1,12 @@
-Einladung zur Mitarbeit beim Lager {{ camp_name }}
+Einladung zur Mitarbeit beim Lager "{{ camp_name | raw }}"
```

Comment:
This is the plain text email, so we don't want to HTML-escape this at all, because it's interpreted as text and not as html by the email clients.

**Test title:**
![image](https://user-images.githubusercontent.com/449555/198863484-7fc88db6-b8d5-466c-8459-8b784b477d3d.png)

**Raw / without escape (as in the PR):**

![image](https://user-images.githubusercontent.com/449555/198863549-9e6ad685-ff79-473d-aa82-bf29adf49020.png)



**With escape:**

![image](https://user-images.githubusercontent.com/449555/198863532-64d1e59e-8743-4ecd-9691-7de4f05b5702.png)